doc/modules/ensemble.rst

```diff
@@ -456,7 +456,7 @@ trees.
   Scikit-learn 0.21 introduces two new experimental implementations of
   gradient boosting trees, namely :class:`HistGradientBoostingClassifier`
   and :class:`HistGradientBoostingRegressor`, inspired by
-  `LightGBM <https://github.com/Microsoft/LightGBM>`_.
+  `LightGBM <https://github.com/Microsoft/LightGBM>`__.
```

Comment:
I did not know double underscores were needed for inline links. Is this a new-ish syntax or has it always been this way? Anyway the links seem to work in the Circle CI rendered HTML.

Reply:
I just read the discussion in sphinx-doc/sphinx#3921. Sorry for the noise.